content/post/107-nix-ipfs-milestone-1.md

```diff
@@ -0,0 +1,111 @@
+---
+date: 2020-08-20
+url: /2020-08-20-nix-ipfs-milestone-1/
+title: Nix × IPFS: Introduction and Milestone 1 Report
+author: John Ericson
+tags: ipfs, nix
```

Comment:
```suggestion
tags: ipfs, nix
header_image: 107-nix-ipfs.jpg
```

Reply:
I added a special header image for this post! 
![107-nix-ipfs](https://user-images.githubusercontent.com/618519/92369236-994a1900-f0ad-11ea-9acc-9e96100873c3.jpg)